src/components/AreaOfResponsibiltySelector.tsx

```diff
@@ -33,7 +51,13 @@ export const AreaOfResponsibilitySelector = () => {
       const areaPromises = areasOfResponsibilityIds.map(async (orgId) => {
         return await fetchOrganization(orgId);
       });
-      return await Promise.all(areaPromises);
+      const organizations = await Promise.all(areaPromises);
+      const organizationOptions = buildOrganizationOptions(organizations);
+      if (!searchParams.get(TicketSearchParam.ViewingScope)) {
```

Comment:
Denne skal absolutt ikke være her, men sjekker inn allikvel slik at det er klart for test i dag. Skal jobbe videre med denne for å få det ut av useQuery og inn i en hook.